popSimManu.tex

```diff
@@ -428,6 +377,70 @@ \subsection*{The Species Catalog}
 as described in our documentation
 (\url{https://stdpopsim.readthedocs.io/en/latest/development.html#naming-conventions}).
 
+ \subsection*{Simulation engines}
+
+Given the genome data and simulation model descriptions defined within the
+library, it is then straightforward to run standardized simulations
+across a range of organisms. \Stdpopsim has a Python API and a user-friendly
+command line interface, allowing users with minimal experience direct access to
+state-of-the-art simulations. Simulations are output in the ``tree sequence''
+format \citep{kelleher2016efficient,kelleher2018efficient,kelleher2019inferring}, which
+contains complete genealogical information about the simulated samples, is
+extremely compact, and can be processed efficiently using the \tskit library
+\citep{kelleher2016efficient,kelleher2018efficient}.
+The tree sequence format could also be converted
+to other formats (e.g., VCF) by the user if desired.
+Currently, \stdpopsim uses the  \texttt{msprime} coalescent simulator \citep{kelleher2016efficient}
+as the default simulation engine.
+
+Coalescent simulations, while highly efficient, are limited in their ability to model
+complex selection scenarios, such as recurrent sweeps and background selection. For these reasons
+we have also implemented the forward-time simulator, \texttt{SLiM} \citep{haller2019tree,haller2019slim}, 
+as an alternative backend engine to \texttt{stdpopsim}, allowing for the simulation of processes that cannot be modeled under 
+the coalescent. However, as forward-time simulators explicitly model all individuals in a population, simulating large
+population sizes can be highly demanding of computational resources. One common practice used to address this challenge is to
+scale down the population size used in the simulations (\textit{N}),
+and then scale down times specified in the demographic model by the same factor, and scale up the mutation and recombination
+rates and selection coefficients such that the scaled parameters, \textit{N$\mu$}, \textit{Nr},
+and \textit{Ns}, remain unchanged \citep[see the \texttt{SLiM} manual and][]{uricchio2014robust}.
+Such down-scaled simulations are not completely equivalent to simulating 
+all individuals in the population, and may lead to subtle differences.
+However, since many sequence-based measures of population diversity remain unchanged when rescaling in this fashion,
+this practice is very common when using forward-time simulators, such as \texttt{SLiM}.
+
+We validated our implementation of the \texttt{SLiM} engine by comparing
+estimates of several population genetic summary statistics for neutral simulations generated by both \texttt{SLiM}
+and \texttt{msprime} under different down-scaling factors
+(Figures \ref{fig:slim_val_map} and \ref{fig:slim_val_nomap}). This comparison indicates that scaling down the simulated
```

Comment:
A lot more validation was done than indicated by these two plots (more summary statistics, and for more demographic models). Maybe the language could be adjusted to reflect that?

Reply:
Could you suggest the change? Or even do it and push to this pr?

Reply:
Suggested edit:
We validated our implementation of the \texttt{SLiM} engine by comparing estimates of several population genetic summary statistics for neutral simulations generated by both \texttt{SLiM} and \texttt{msprime} under various demographic models and using different down-scaling factors. We show a summary of two such comparisons for the AncientEurasia\_9K19 model \citep{kamm2019efficiently} using the
HapMapII\_GRCh37 genetic map \citep{international2007second} and a uniform recombination rate (Figures \ref{fig:slim_val_map} and \ref{fig:slim_val_nomap}, resp.).
These comparisons indicate ...